src/main/kotlin/com/careem/gradle/dependencies/main.kt

```diff
@@ -38,14 +39,19 @@ class Tldr : CliktCommand(name = "dependency-diff-tldr") {
     help = "Collapse packages with a matching group under a group.*. Collapsing " +
         "will only occur if all version numbers match. (ex --collapse com.careem.ridehail --collapse com.careem.now)."
   ).multiple()
+  private val output by option(
+    "-o",
+    "--output",
+    help = "Output type, \"plain\" and \"json\" are supported"
+  ).default("plain")
```

Comment:
TODO I'm not able to run the script end to end locally @ahmedre could you support me?

Reply:
#21 solves this, thanks for pointing it out